frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1447,550 +1876,150 @@ function ReaderFooter:addToMainMenu(menu_items)
-    end
-    table.insert(sub_items, getMinibarOption("bookmark_count"))
-    table.insert(sub_items, getMinibarOption("percentage"))
-    table.insert(sub_items, getMinibarOption("book_time_to_read"))
-    table.insert(sub_items, getMinibarOption("chapter_time_to_read"))
-    if Device:hasFrontlight() then
-        table.insert(sub_items, getMinibarOption("frontlight"))
+        table.insert(configure_complications_sub_table , 5, {
+            text_func = function()
+                if self.settings.battery_hide_threshold <= (Device:hasAuxBattery() and 200 or 100) then
+                    return T(_("Hide battery item when higher than: %1".. "%"), self.settings.battery_hide_threshold)
+                else
+                    return _("Hide battery item at custom threshold")
+                end
+            end,
+            checked_func = function()
+                return self.settings.battery_hide_threshold < MAX_BATTERY_HIDE_THRESHOLD
+            end,
+            enabled_func = function()
+                return self.settings.all_at_once == true
+            end,
+            separator = true,
+            callback = function(touchmenu_instance)
+                local SpinWidget = require("ui/widget/spinwidget")
+                local battery_threshold = SpinWidget:new{
+                    value = math.min(self.settings.battery_hide_threshold, Device:hasAuxBattery() and 200 or 100),
+                    value_min = 0,
+                    value_max = Device:hasAuxBattery() and 200 or 100,
+                    default_value = Device:hasAuxBattery() and 200 or 100,
+                    unit = "%",
+                    value_hold_step = 10,
+                    title_text = _("Set minimum threshold to hide battery item"),
+                    callback = function(spin)
+                        self.settings.battery_hide_threshold = spin.value
+                        self:refreshFooter(true, true)
+                        if touchmenu_instance then touchmenu_instance:updateItems() end
+                    end,
+                    extra_text = _("Disable"),
+                    extra_callback = function()
+                        self.settings.battery_hide_threshold = MAX_BATTERY_HIDE_THRESHOLD
+                        if touchmenu_instance then touchmenu_instance:updateItems() end
+                    end,
+                    ok_always_enabled = true,
+                }
+                UIManager:show(battery_threshold)
+            end,
+            keep_menu_open = true,
+        })
     end
-    if Device:hasNaturalLight() then
-        table.insert(sub_items, getMinibarOption("frontlight_warmth"))
+    ----------- More status bar options
+    -- quick access to this setting for "@NiLuJe, and for people that do like him." -- poire-z (2024)
+    table.insert(sub_items, getMinibarOption("reclaim_height"))
+    table.insert(sub_items, {
+        text = _("Show status bar divider"),
```

Comment:
I think `separator` is better

Reply:
Dunno if separator is better, may be.
But seeing ![image](https://github.com/koreader/koreader/assets/24273478/ae5973a5-b884-42be-b55a-f106e15d6e5c) and having already forgotten what this does, I had to try it to guess what it does.
No real better idea ("Show separator/horizontal lines above status bar" ...)